src/pages/ScholarshipForm.jsx

```diff
@@ -53,23 +41,22 @@ function ScholarshipForm() {
         .save()
         .then((scholarship) => {
           setSubmissionAlert(
-            showSubmissionAlert(
-              'success',
-              `${scholarship.data.name} submitted successfully.`
-            )
+            <Alert onClose={() => setSubmissionAlert(null)} severity="success">
+              <AlertTitle>Success</AlertTitle>
+              {`${scholarship.data.name} submitted successfully.`}
+            </Alert>
           );
           resetForm();
         })
-        .catch(() =>
+        .catch((error) =>
           setSubmissionAlert(
-            showSubmissionAlert(
-              'error',
-              'Something went wrong. Please try again later.'
-            )
+            <Alert onClose={() => setSubmissionAlert(null)} severity="error">
+              <AlertTitle>Error</AlertTitle>
+              {error}
```

Comment:
Have you tested this? You might need to do `error.toString()`

Reply:
yes, my bad.